ci/src/dependencies/job/npm_scanner_periodic_job.py

```diff
@@ -97,7 +97,7 @@
                 owner=Team.GIX_TEAM,
             )
         ],
-        DEFAULT_NODE_VERSION,
+        "22.11.0",
```

Comment:
I would either update `DEFAULT_NODE_VERSION` or delete this var entirely to avoid any confusion.